app/scripts/controllers/registercontainer.js

```diff
@@ -109,7 +111,7 @@ angular.module('dockstore.ui')
       };
 
       $scope.getImagePath = function(imagePath, part) {
-        var imagePathRegexp = /^(([a-zA-Z0-9]+([-_][a-zA-Z0-9]+)*)|_)\/([a-zA-Z0-9]+([-_][a-zA-Z0-9]+)*)$/i;
+        var imagePathRegexp = /^(([a-zA-Z0-9]+([-_.][a-zA-Z0-9]+)*)|_)\/([a-zA-Z0-9]+([-_.][a-zA-Z0-9]+)*)$/i;
```

Comment:
I know this wasn't introduced in this branch, but this is getting a bit long. 
Please add a comment explaining what this regex is attempting to match.